### DISCUSSION TITLE
Add option to launch maximized?

### DISCUSSION THREAD

**Discussion by adm193** [mifi/lossless-cut#1647] [Ideas]
Whenever I open lossless cut it is in a very small window and I have to maximize it every time. It would be nice if there was an option to toggle this on or off depending on who would want it.

**Comment by mifi** [comment]
Which version are you on? I believe what you're asking for is already implemented, see #44

**Comment by adm193** [comment]
I am on 3.54.0.0. Thank you for linking that thread, I have resized my window accordingly to open larger every time. I was mostly wondering if there can be an option implemented to launch the program in a maximized state. For now, though, I am happy to at least be able to resize the window to get close to a "full" maximized state. Thanks!

**Reply by mifi** [reply]
It's the first time I've heard the full-screen launch request. I will consider adding it if more people are requesting it, thanks!
